packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -29,6 +33,7 @@ const metadata: Meta = {
         statusLink: 'https://github.com/ni/nimble/issues/296'
     })}
         <${cardTag}>
+            <span slot="title">${x => x.title}</span>
```

Comment:
In the storybook docs page you'll notice that the select pop-up does not pop all the way out like you would expect and looks stuck.

![image](https://github.com/ni/nimble/assets/1588923/208997f1-99d1-428e-a172-8b51dfd26a3d)

This is a known issue and you can fix it by using this helper at the start of the story template:

https://github.com/ni/nimble/blob/f159a084527c7e53bac99dd43182c10c2f2f628f/packages/nimble-components/src/combobox/tests/combobox.stories.ts#L53

And looks like it causes issues in chrome now too! They might be behaving more consistently. Can you update the line in the helpers to remove mentions of specific browsers?

https://github.com/ni/nimble/blob/f159a084527c7e53bac99dd43182c10c2f2f628f/packages/nimble-components/src/utilities/tests/storybook.ts#L166

Also, handy fyi, if you are trying to debug (or in my case take a snippet) of an element that is only shown on hover / focus, you can use the [emulate a focused page](https://developer.chrome.com/docs/devtools/rendering/apply-effects/#emulate-a-focused-page) feature to keep them visible when clicking outside the window

Reply:
Added `disableStorybookZoomTransform` to the card storybook and updated the comment in the utility to remove the mention of Firefox.

>Also, handy fyi, if you are trying to debug (or in my case take a snippet) of an element that is only shown on hover / focus, you can use the [emulate a focused page](https://developer.chrome.com/docs/devtools/rendering/apply-effects/#emulate-a-focused-page) feature to keep them visible when clicking outside the window

This will be super useful, thanks for this!